### PR TITLE
Add recipe for E-ANTIC

### PR DESCRIPTION
**Pull request by saraedum**
E-ANTIC is a C/C++ library for mathematical software that provides
approximate and exact arithmetic for elements of real embedded number fields.

### CODE REVIEW

**Comment by saraedum** [comment]
cc @videlec

**Comment by saraedum** [comment]
@conda-forge/help-c-cpp I think this recipe is ready for review :)

File: recipes/e-antic/build.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+export CFLAGS="-O2 -g $CFLAGS"
+export CXXFLAGS="-O2 -g $CXXFLAGS"
+export LD_LIBRARY_PATH="${PREFIX}/lib:${LD_LIBRARY_PATH}"
+export DYLD_FALLBACK_LIBRARY_PATH="${PREFIX}/lib:${DYLD_FALLBACK_LIBRARY_PATH}"
```
Comment (xhochy) [inline recipes/e-antic/build.sh]: All four export lines should not be required. Is something failing without them?

Comment (saraedum) [reply recipes/e-antic/build.sh]: Thanks for the feedback :) I had to put these in other recipes in the past. Let me try what happens if I remove them.

Comment (saraedum) [reply recipes/e-antic/build.sh]: @xhochy: Guess you were right. Looks good to me without these lines.

Comment (xhochy) [reply recipes/e-antic/build.sh]: With the recent migration to the new compilers most of these flags are now already set automatically. Also, the `LD_LIBRARY_PATH` related things should be automatically correct by `conda` itself.

**Comment by scopatz** [comment]
@videlec - are you willing to be a maintainer here?

**Comment by videlec** [comment]
@scopatz Sure. I should be able to update the conda package when new releases come out.

**Pull request merged**